### DISCUSSION TITLE
fatal: unable to look up git@gitlab.com:repo-name (port 9418) (Name does not resolve)

### DISCUSSION THREAD

**Discussion by mikaelparkefelt** [semaphoreui/semaphore#1978] [Q&A] [answered]
I have task template and when I run it I get this output:

```
Cloning Repository git://git@gitlab.com:group-name/repo-name.git
Cloning into 'repository_15_7'...
fatal: unable to look up git@gitlab.com:repo-name (port 9418) (Name does not resolve)
Failed updating repository: exit status 128
Running playbook failed: exit status 128
```

What is the problem here and how do I resolve it?

**Comment by fiftin** [comment, marked as answer]
Hi @mikaelparkefelt ,

I didn't know that GitHub supports git:// protocol. Please try ssh://.

**Reply by mikaelparkefelt** [reply]
Now it is working!!?!?!?!?!?!?
I don't now what I have changed but I did not change anything....
It must have been a glitch....
